.github/workflows/deploy-stack.yml

```
@@ -96,7 +96,7 @@ jobs:
       - name: Setup Python
```

Please could it go in the name, too?

```suggestion
      - name: Setup Python 3.14
```